acra-censor/handlers/blacklist_handler.go

```diff
@@ -45,21 +46,24 @@ func (handler *BlacklistHandler) CheckQuery(query string) (bool, error) {
 				case *sqlparser.AliasedTableExpr:
 					err = handler.handleAliasedTables(fromStatement.(*sqlparser.AliasedTableExpr))
 					if err != nil {
-						log.WithError(err).Debugln("error from BlacklistHandler.handleAliasedTables")
+						log.WithError(err).Debugln("acra-censor: error from BlacklistHandler.handleAliasedTables")
+						log.WithError(ErrAccessToForbiddenTableBlacklist)
```

Comment:
`log.WithError(err)` == `logger := log.WithField("error", err)` and do nothing. It's just return new *log.Entry with predefined field `error`.